app/db/index.js

```diff
@@ -1,40 +1,50 @@
-const firebase = require('firebase');
-const config = require('../config');
+const MongoClient = require('mongodb').MongoClient;
+const uri = "mongodb+srv://tseucsd:abctse@live-weller-5tgkd.mongodb.net/admin?retryWrites=true&w=majority";
 
-if (!firebase.apps.length) {
-  firebase.initializeApp(config.firebase);
-}
+let db = null;
+setupReference().then(ref => db = ref);
 
-const db = firebase.database().ref('live_weller').child('markets');
+/**
+ * Call this method to return a Promise to retrieve a reference to MongoDB.
+ */
+function setupReference() {
+    return new Promise((resolve, reject) => {
+        MongoClient.connect(uri, { useUnifiedTopology: true, useNewUrlParser: true })
+            .then(result => resolve(result))
+            .catch(function(err) { console.log(err); })
+    });
+}
 
 /**
- * By calling this method and using .then() for the callback, you can access
+ * By calling this method and using .then() for the callback, you can access 
  * the entire list of markets in JSON format.
- *
+ * 
  * This returns a Promise so that once the database values are secured, they can
  * be worked with in a different location/file.
  */
 function getAllMarkets() {
-  return new Promise((resolve, reject) => {
-    db.once('value')
-      .then((result) => resolve(result.val()))
-      .catch((err) => reject(err));
-  });
+    return new Promise((resolve, reject) => {
+        db.db('live-weller-test').collection('markets')
```

Comment:
change the database name and configure it in the `.env` (and add it as a variable in `/app/config`)